### PR TITLE
chore: upgrade deps and fix test

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,21 +23,20 @@
     "app.js"
   ],
   "devDependencies": {
-    "autod": "^2.7.0",
+    "autod": "^2.7.1",
     "egg": "test",
```
Comment (atian25) [inline package.json]: 改为 ^0.9.0?

File: package.json [hunk 2]
```diff
@@ -23,21 +23,20 @@
...
-    "eslint": "^3.3.1",
-    "eslint-config-egg": "^3.1.0",
-    "glob": "^7.0.5",
+    "egg-bin": "^2.0.2",
+    "egg-ci": "^1.1.0",
+    "egg-mock": "^2.3.1",
+    "eslint": "^3.14.1",
+    "eslint-config-egg": "^3.2.0",
+    "glob": "^7.1.1",
     "intelli-espower-loader": "^1.0.1",
```
Comment (atian25) [inline package.json]: `intelli-espower-loader` 干掉

File: package.json [hunk 3]
```diff
@@ -23,21 +23,20 @@
...
     "pedding": "1",
-    "power-assert": "^1.4.1",
-    "supertest": "^2.0.0"
+    "supertest": "^3.0.0"
   },
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=6.0.0"
   },
   "scripts": {
```
Comment (atian25) [inline package.json]: 这几条 scripts 需要都改改

**Comment by popomore** [comment]
改了

**Comment by popomore** [comment]
https://github.com/eggjs/egg/issues/309

**Comment by popomore** [comment]
测试过了

**Pull request merged**